skgstat/Variogram.py

```diff
@@ -801,13 +825,21 @@ def n_lags(self, n):
         # if there are no errors, store the passed value
         self._n_lags_passed_value = n
 
-        # reset the groups
+        # reset the groups and count
         self._groups = None
+        self._count = None
 
         # reset the fitting
         self.cof = None
         self.cov = None
 
+    @property
+    def count(self):
```

Comment:
I like your approach very much, but I am just wondering if `bin_count` would make a better property name, to make things more clear. What do you think?

The 'internal' `_count` is fine, but could also go with `_bin_count` to make the relationship more clear

Reply:
Changed to `bin_count` in 0e80abf
All tests passing!